packages/x-date-pickers/src/PickersLayout/PickersLayout.tsx

```diff
@@ -73,11 +73,19 @@ export const PickersLayoutContentWrapper = styled('div', {
   name: 'MuiPickersLayout',
   slot: 'ContentWrapper',
   overridesResolver: (props, styles) => styles.contentWrapper,
-})({
+})<{ ownerState?: { valueType: ValueType } }>({
```

Comment:
I would prefix this one in case we want to add it to the main `ownerState` later on.

```suggestion
})<{ ownerState?: { pickersValueType: ValueType } }>({
```

I would add this property in `usePickerLayout` and return the `ownerState` from there, that way we use the exact same object in everything layout-related.
Note that this change is a breaking change since people are using this DOM element when building custom layout.


____

With that being said, I find the change quite fragile, not sure how we could improve it...
I think we should be able to use something like `grid-column: 1/-1;` to avoid hardcoding the number of columns on items that should spread from left to right.